pkg/localMetrics/localMetrics.go

```diff
@@ -0,0 +1,16 @@
+package localMetrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	MetricTotalAWSAccounts = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
I think you can just use `NewGauge` here. Can you also rename this metric to DummyMetric?